python-regression/tests/features/machine1/1_api_tests.feature

```diff
@@ -154,7 +154,7 @@ Feature: Test API calls on Machine 1
 
 		Then the response for "getInclusionStates" should return with:
 		|keys			|values			|type               |
-		|states			|False		        |bool               |
+		|states			|True		        |bool               |
```

Comment:
really?
We will wait for @DyrellC to say whether this makes sense and wait for buildkite

Reply:
ok

Reply:
Btw, @DyrellC, this is what I was talking to you about needing to to properly document the DBs, currently it changed and I have no idea why by just looking at the code.

Besides that, @acha-bill, maybe it is worth while to have another scenario (or step) in this PR for `getInclusionStates`. We want to have both the `True` and `False` cases. 
We also want to make sure that we can accept a list of Hashes and not just one single hash.
You can wait for @DyrellC to explain to you what is going on with the DB

Reply:
@galrogo this instance was actually the db that you had provided in the previous regression tests, but it does highlight this point yes. 
@acha-bill since the `getInclusionStates` call will no longer need the tip list, you can remove that from the input side of the test. You could also add the list example that @galrogo is asking for with a 2 for 1 by having another scenario that tests inclusion state on a list of randomly generated hashes and having that return a `boolList` of false. That would cover testing a list scenario as well as making sure we have false cases register false as well.

Reply:
@DyrellC The python lib throws `expected 3 args` for `getInclusionStates`.
So we'll leave the `tips` param there for now.

Reply:
Hmm I am adding communicating with clients to checklist